JASP-Tests/R/tests/testthat/test-anova.R

```diff
@@ -201,8 +207,8 @@ test_that("Q-Q plot matches", {
   options$fixedFactors <- "contBinom"
   options$modelTerms <- list(list(components="contBinom"))
   options$qqPlot <- TRUE
-  results <- jasptools::run("Anova", "test.csv", options)
-  testPlot <- results[["state"]][["figures"]][[1]][["obj"]]
+  results <- jasptools::run("Anova", "test.csv", options, view = FALSE)
```

Comment:
```suggestion
  results <- jasptools::run("Anova", "test.csv", options)
```
view argument is redundant (here and in the other calls).